ui/components/app/flask/snap-remove-warning/index.scss

```diff
@@ -0,0 +1,25 @@
+.snap-remove-warning {
+  color: black;
+
+  .popover-header {
+    padding: 24px 16px 0px 24px;
+  }
+
+  &__content {
+    padding: 0px 16px 16px 24px;
+    align-items: center;
+
+    .typography--h6 {
+      font-size: 18px;
```

Comment:
[req] The Typography component has font styles set using props, what you are looking for is supplying the 'variant' prop, and 18px is 1.125 rem which equates to `variant={TYPOGRAPHY.H4}` on the Typography component